csharp/src/Google.Protobuf.Benchmarks/Google.Protobuf.Benchmarks.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
why specifically netcoreapp2.2?

Reply:
I'm expecting that to be more interesting than netcoreapp1.0, which is basically ancient. If we're looking at improving performance with modern APIs, I'm most interested in testing in modern runtimes too.

That said, I certainly wouldn't object to using multiple target frameworks. We can't target netcoreapp1.0 though with the current version of BenchmarkDotNet.